project/dexter-monitor/public/ctrl/currentProjectCtrl.js

```diff
@@ -0,0 +1,93 @@
+        {field:'defectCountTotal',      displayName:'Total',            width: '9%',    cellClass: 'grid-align',
+            headerTooltip: 'Number of all defects', aggregationType: uiGridConstants.aggregationTypes.sum},
+        {field:'defectCountFixed',      displayName:'Fixed',            width: '9%',    cellClass: 'grid-align',
+            headerTooltip: 'Number of fixed defects', aggregationType: uiGridConstants.aggregationTypes.sum},
+        {field:'defectCountDismissed',  displayName:'Dismissed',        width: '9%',    cellClass: 'grid-align',
+            headerTooltip: 'Number of dismissed defects', aggregationType: uiGridConstants.aggregationTypes.sum},
+        {field:'resolvedRate',          displayName:'Resolved rate',    width: '11%',   cellClass: 'grid-align',
+            headerTooltip: '(Fixed + Dismissed) / Total * 100', footerCellClass: 'grid-align',
+            aggregationType: () => `${$scope.resolvedRateTotal}`,
+            cellTemplate:'<div class="ui-grid-cell-contents">{{grid.appScope.getResolvedRate(row.entity)}}</div>'},
+        {field:'serverStatus',          displayName:'Server Status',    width: '16%',   cellClass: 'grid-align',
+            headerTooltip: 'Server status', footerCellClass: 'grid-align',
+            aggregationType: () => `Active: ${$scope.activeServerCount} / Inactive: ${$scope.allServerCount-$scope.activeServerCount}`,
+            cellTemplate:'<div class="ui-grid-cell-contents"' +
+                        ' ng-class="{\'server-status-active\':COL_FIELD == \'Active\',' +
+                        '            \'server-status-inactive\':COL_FIELD == \'Inactive\'}">{{COL_FIELD}}</div>'}
+    ];
+
+    initialize();
+
+    function initialize() {
+        $scope.resolvedRateTotal = '';
+        $scope.allServerCount = 0;
+        $scope.activeServerCount = 0;
+        $scope.gridOptions = createGrid(columnDefs);
+        $scope.gridOptions.showColumnFooter = true;
+        ServerStatusService.getActiveServerList()
+            .then(loadData)
+            .catch((err) => {
+                $log.error(err);
+            });
+        $scope.time = new Date().toLocaleString();
+        setGridExportingFileNames($scope.gridOptions, CURRENT_STATUS_FILENAME_PREFIX + '-' + $scope.time);
+    }
+
+    function loadData(activeServerList) {
+        ProjectService.getAllCurrentStatusList(activeServerList)
+            .then((rows) => {
+                $scope.gridOptions.data = rows;
+                const defectCountTotalSum = _.sum(_.map($scope.gridOptions.data, 'defectCountTotal'));
+                const defectCountFixedSum = _.sum(_.map($scope.gridOptions.data, 'defectCountFixed'));
+                const defectCountDismissedSum = _.sum(_.map($scope.gridOptions.data, 'defectCountDismissed'));
+                $scope.resolvedRateTotal = `${((defectCountFixedSum + defectCountDismissedSum) / defectCountTotalSum * 100).toFixed(1)}%`;
```

Comment:
`NaN` is displayed. I'll change this code so that no content is displayed when `defectCountTotalSum` is zero.